README.md

```diff
@@ -22,7 +24,58 @@ This token will need:
 * Read/Write access to Linodes (to attach and detach volumes)
 * A sufficient "Expiry" to allow for continued use of your volumes
 
-### Create a Kubernetes secret
+### [Recommended] Using Helm to deploy the CSI Driver
+
+Use the helm chart located under './helm-chart/csi-driver'. This dir has the manifest for Linode Block Storage CSI Driver. There are two arguments required.
+
+The first argument must be a Linode APIv4 [Personal Access Token](https://cloud.linode.com/profile/tokens) with all permissions.
+
+The second argument must be a Linode [region](https://api.linode.com/v4/regions).
+
+#### To deploy the CSI Driver run the following:
+```sh
+git clone git@github.com:linode/linode-blockstorage-csi-driver.git
+
+cd linode-blockstorage-csi-driver
+
+helm install linode-csi-driver ./helm-chart/csi-driver --set apiToken=$LINODE_API_TOKEN,region=$REGION
+```
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+#### Uninstall
+
+To uninstall linode-csi-driver from kubernetes cluster. Run the following command:
+
+```sh
+
+helm uninstall linode-csi-driver
+
+```
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+#### Upgrade
+
+To upgrade when new changes are made to the helm chart. Run the following command:
+
+```sh
+
+helm upgrade linode-csi-driver ./helm-chart/csi-driver --install
```

Comment:
We need to specify token and region as well as its required for upgrade as well.